homeassistant/components/geo_json_events/geo_location.py

```diff
@@ -104,7 +104,10 @@ async def async_update(self) -> None:
 
     def _update_from_feed(self, feed_entry: GenericFeedEntry) -> None:
         """Update the internal state from the provided feed entry."""
-        self._attr_name = feed_entry.title
+        if feed_entry.properties and "name" in feed_entry.properties:
+            self._attr_name = feed_entry.properties.get("name")
```

Comment:
If `feed_entry.properties` is a dict, please use `dict[key]` to access a known existing item.

Reply:
you are checking if it exists in the if statement its in ;)

Reply:
Right, we've already checked that it exists and has a key `"name"`. So there's no need to use `dict.get(key)`.

Reply:
(I must say I also didn't spot this, good catch)